web/static/css/components/_AchievementList.scss

```diff
@@ -0,0 +1,4 @@
+.AchievementList {
+  list-style: none;
```

Comment:
Properties should be ordered padding, list-style

[Read more about it here.](https://github.com/brigade/scss-lint/blob/master/lib/scss_lint/linter/README.md#propertysortorder)